index.html

```diff
@@ -0,0 +1,197 @@
+            </h2>
+            <ul>
+                <li></li>
+                <li></li>
+                <li></li>
+                <li></li>
+                <li></li>
+            </ul>
+            <div class="search"></div>
+        </div>
+    </header>
+    <main class="main">
+        <div class="content">
+            <h1><span class="center-text">Some text here</span></h1>
+            <article class="story">
+                <div class="author">
+                    <p>James</p>
+                </div>
+                <div class="article-title">
+                    <p>Understanding CSS</p>
+                </div>
+                <div class="tags">
+                    <p>#something</p>
+                </div>
+            </article>
+
+            <article class="story light">
+                <div class="author">
+                    <p>James</p>
+                </div>
+                <div class="article-title">
+                    <p>Understanding CSS</p>
+                </div>
+                <div class="tags">
+                    <p>#something</p>
+                </div>
+            </article>
+
+            <article class="story dark">
+                <div class="author">
+                    <p>James</p>
+                </div>
+                <div class="article-title">
+                    <p>Understanding CSS</p>
+                </div>
+                <div class="tags">
+                    <p>#something</p>
+                </div>
+            </article>
+
+            <article class="story">
+                <div class="author">
+                    <p>James</p>
+                </div>
+                <div class="article-title">
+                    <p>Understanding CSS</p>
+                </div>
+                <div class="tags">
+                    <p>#something</p>
+                </div>
+            </article>
+        </div>
+        <div class="blue">
+            <div class="blue-button">
+
+            </div>
+        </div>
+        <div class="four-divs">
+            <section>
+            <h2>Title</h2>
+            </section>
+            <section>
+          <h2>Title</h2>
+            </section>
+            <section>
+            <h2>Title</h2>
+            </section>
+            <section>
+            <h2>Title</h2>
+            </section>
+        </div>
+        <div class="content red">
+            
+            <article class="red-square">
+                <div class="explore-features"></div>
+            </article>
+
+            <article class="twin-parent">
+                <div class="twincolumns"></div>
+                <div class="twincolumns"></div> 
+            </article>
+
+            <article class="twin-parent">
+                <div class="twincolumns"></div>
+                <div class="twincolumns"></div>
+            </article>
+
+            <article class="red-square">
+                <div class="explore-features"></div>
+            </article>
+        </div>
```

Comment:
Kindly, make this part have the remaining 2 rows with 3 articles per row like this
![kro5](https://user-images.githubusercontent.com/46388113/74501263-7eca0b80-4ee1-11ea-81c7-ead30a100f8d.PNG)
![kro6](https://user-images.githubusercontent.com/46388113/74501274-8689b000-4ee1-11ea-838e-68d5933f9cf2.PNG)